### PR TITLE
Display error details, if present

### PR DESCRIPTION
**Pull request by baijum**
Also, add a link to the error log.
    
This is a follow-up PR on #69

Here is an example comment:

pr-comment

### CODE REVIEW

File: scripts/prepare_pr_comment.py [hunk 1]
```diff
@@ -1,8 +1,18 @@
+import os
 import sys
 
 def prepare_failure_commet(repository, issue_number, vendor_label, chart_name):
```
Comment (pedjak) [inline scripts/prepare_pr_comment.py:4]: typo - `prepare_failure-comment`

File: scripts/prepare_pr_comment.py [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+    runid = open("./pr/build-verify-check-run-id").read()
+    run_url = f"https://github.com/{repository}/pull/{issue_number}/checks?check_run_id={runid}".strip()
+    msg += f'There are some errors while building and validating your PR.'
+    msg += 'Please check the log for more details:\n'
+    msg += run_url
+    msg += '\n(Please open this link in another tab/window)\n\n'
+    if os.path.exists("./pr/errors"):
+        msg += f"These are the critical issues:\n\n"
+        msg += open("./pr/errors").read()
+    msg += "\n---\n"
```
Comment (pedjak) [inline scripts/prepare_pr_comment.py:15]: would you mind to switch to multiline string, they read better IMHO. For example:

msg = f"""
   Thank you for the PR #{issue_number}!

   There are some errors while building and validating your PR.

"""

Comment (mmulholla) [reply scripts/prepare_pr_comment.py:15]: Notice mixed use of single quote and double quote, if not need for some python reason consider using one or the other.

File: scripts/prepare_pr_comment.py (hunk 1 above)
Comment (baijum) [reply scripts/prepare_pr_comment.py]: Done. 587b3a34a14d16486d54d829c161766b9ee1f9eb

File: scripts/prepare_pr_comment.py (hunk 2 above)
Comment (baijum) [reply scripts/prepare_pr_comment.py]: Done. 562c786b9c7ff2c7caf66d63f731dbff939e43f1

File: scripts/prepare_pr_comment.py [hunk 3]
```diff
@@ -1,13 +1,34 @@
+import os
 import sys
 
-def prepare_failure_commet(repository, issue_number, vendor_label, chart_name):
-    msg = f"Thank you for the PR #{issue_number}!\n\n"
-    msg += f"There are [some errors](https://github.com/{repository}/pull/{issue_number}/checks) while building and validating your PR. Please check the log for more details.\n\n"
-    msg += f'/metadata {{"vendor_label": "{vendor_label}", "chart_name": "{chart_name}"}}\n\n'
-    msg += f"Partner Support URL: https://redhat-connect.gitbook.io/red-hat-partner-connect-general-guide/managing-your-account/getting-help/technology-partner-success-desk\n"
```
Comment (dperaza4dustbit) [inline scripts/prepare_pr_comment.py:7]: This messages do not match the test you posted here. Can you run the test again to see this message format?

**Review by dperaza4dustbit — COMMENTED** [review]
Looks good but test output and current message do not match.

File: scripts/prepare_pr_comment.py (hunk 3 above)
Comment (baijum) [reply scripts/prepare_pr_comment.py:7]: These red lines are removed. See below for new text.

File: scripts/prepare_pr_comment.py [hunk 4]
```diff
@@ -1,13 +1,34 @@
...
+
+{errors}
+"""
+
+    msg += f"""
+---
+/metadata {{"vendor_label": "{vendor_label}", "chart_name": "{chart_name}"}}
+
+Partner Support: http://bit.ly/technology-partner-success-desk
+"""
```
Comment (dperaza4dustbit) [inline scripts/prepare_pr_comment.py:28]: These look good.

**Review by dperaza4dustbit — APPROVED** [review]
+1

**Pull request merged**